ui/src/components/ObjectConfiguration/index.tsx

```diff
@@ -31,7 +31,9 @@ const ObjectConfiguration: React.FC<ObjectConfigurationProps> = ({
 
   const spec = inNode ? config : config.kube_object.spec
   const experiment =
-    inSchedule || inNode ? spec[toCamelCase(inSchedule ? spec.type : (config as any).templateType)] : spec
+    inSchedule || inNode
+      ? spec[mappingTemplateType2FieldName(inSchedule ? spec.type : (config as any).templateType)]
```

Comment:
Should be `mappingTemplateType2FieldName.get`.

Reply:
I also think should give it a meaningful name, like `kindToTemplateType`?

Reply:
> Should be `mappingTemplateType2FieldName.get`.

`mappingTemplateType2FieldName` is the name of function, it call `mapping.get()` inside.

Reply:
> I also think should give it a meaningful name, like `kindToTemplateType`?

How about `TemplateTypeToFieldName`?

Reply:
Oh, got it. I miss the function definition. `TemplateTypeToFieldName` is OK.

Reply:
Oh @g1eny0ung  which should I use, `TemplateTypeToFieldName` or `templateTypeToFieldName`?

Reply:
The latter. `camelCase` is the customary usage of JS.

Reply:
updated PTAL @g1eny0ung